specification/resources/resource-manager/Microsoft.Resources/stable/2019-10-01/resources.json

```diff
@@ -285,6 +286,12 @@
               "$ref": "#/definitions/DeploymentValidateResult"
             }
           },
+          "202": {
+            "description": "Accepted - The request has been accepted for processing and the operation will complete asynchronously.",
+            "schema": {
+              "$ref": "#/definitions/DeploymentValidateResult"
```

Comment:
Validate response or empty?  The other 202s seem to have no schema listed.

Reply:
No schema needed. Updated in the last iteration. 

---
In reply to: [406609375](https://github.com/Azure/azure-rest-api-specs/pull/9007#discussion_r406609375) [](ancestors = 406609375)